server_transcriptor/external/secret.yaml

```diff
@@ -0,0 +1,2 @@
+key: 8ae4c6464f7d42b19c690b7e3c10693e
```

Comment:
should this be here?

Reply:
it shouldn't but it's just a trial account so nobody can really exploit it and it's easier right now for testing to just have it there